bundles/edu.kit.kastel.sdq.eclipse.grading.view/src/edu/kit/kastel/eclipse/grading/view/assessment/ArtemisGradingView.java

```diff
@@ -347,6 +349,13 @@ private void createAssessmentTab(TabFolder tabFolder) {
 		this.addSelectionListenerForRefreshArtemisStateButton(btnRefreshArtemisState);
 		this.addControlToPossibleActions(btnRefreshArtemisState, Transition.ON_RESET);
 
+		var pluginVersion = Activator.getDefault().getBundle().getVersion();
+		Label version = new Label(assessmentComposite, SWT.NONE);
+		version.setLayoutData(new GridData(SWT.RIGHT, SWT.BOTTOM, true, false, 2, 1));
+		version.setAlignment(SWT.RIGHT);
+		version.setText("Artemis Grading " + //
```

Comment:
String::format would be prettier :)

Reply:
Yes , I'll fix that when I'm at the PC again.